site/content/events/2016-seattle/index.html

```diff
@@ -11,8 +11,7 @@
 <center>
   <table><tr><td>
         <center>
-        <!-- img border=0 width="160" height="240" src="./yourlogohere.png">
--->
+        <!-- <img border=0 width="160" height="240" src="./yourlogohere.png"> -->
```

Comment:
we are having an artist draw up some logos for us to use. So i was just going to leave the logo blank until we get that done in a few weeks.

Thanks.